deploy-agent/deployd/common/helper.py

```diff
@@ -65,17 +65,26 @@ def get_stale_builds(build_timestamps, num_builds_to_retain=2):
     def clean_package(base_dir, builds, build_name):
         local_fn = '{}-{}.*'.format(build_name, builds)
         try:
+            # Remove extracted pointer from disk
             extracted_file = os.path.join(base_dir, '{}.extracted'.format(builds))
             if os.path.exists(extracted_file):
                 os.remove(extracted_file)
+            # Remove staged pointer from disk
             staged_file = os.path.join(base_dir, '{}.staged'.format(builds))
             if os.path.exists(staged_file):
                 os.remove(staged_file)
         except OSError as e:
             log.error(e)
 
         try:
+            # Remove build directory from disk
             shutil.rmtree(os.path.join(base_dir, builds))
+        except BaseException as e::
```

Comment:
BaseException seems too wide since it also include SystemExit, KeyboardInterrupt. Otherwise, it's hard to stop exit the code. A better option might be just Exception.